src/main/scala/cromwell/engine/backend/Backend.scala

```diff
@@ -37,6 +37,7 @@ object Backend {
  * Trait to be implemented by concrete backends.
  */
 trait Backend {
+  type A <: BackendCall
```

Comment:
I might back off of that, but I'll need to wait until I'm not githubbin g on my phone :)